docker/elasticsearch.Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG version
+FROM docker.elastic.co/elasticsearch/elasticsearch:${version}
+ARG version
```

Comment:
You wrote `ARG version` twice.

Reply:
Baha I dun goofed for not seeing that.

Reply:
On purpose, the first one declares the ARG and allows to use it in the FROM.
The second one allows to use it in the rest of the Dockerfile.
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Reply:
Oh, ok, this is not intuitive at all. :)